packages/wonder-blocks-toolbar/components/toolbar.md

```diff
@@ -11,40 +11,34 @@ const secondaryButton = <Button size="small" kind="secondary">Secondary button</
 ### Toolbar with title
 
 ```js
-// TODO(scottgrant): Use wonder-blocks-icon and wonder-blocks-icon-button
-const dismissIcon =
-    "M12.003 10.588L7.707 6.293a1 1 0 0 0-1.414 1.414l4.295 4.296-4.295 4.295a1 1 0 0 0 1.414 1.414l4.296-4.295 4.295 4.295a1 1 0 0 0 1.414-1.414l-4.295-4.295 4.295-4.296a1 1 0 1 0-1.414-1.414l-4.295 4.295z";
-const closeButton = (
-    <svg role="img" width="24px" height="24px">
-        <path d={dismissIcon} />
-    </svg>
-);
+const Color = require("@khanacademy/wonder-blocks-color").default;
+const {icons} = require("@khanacademy/wonder-blocks-icon");
+const IconButton = require("@khanacademy/wonder-blocks-icon-button").default;
+
+const closeButton = <IconButton icon={icons.dismiss} style={{color: Color.offBlack}} />;
```

Comment:
Instead of setting `style={{color: ...}}` use `kind="secondary"` (offBlack) or `kind="tertiary"`.  I think the latter fits better given the mocks of the modals.  The close button is more subtle there.